comit/src/network/protocols/orderbook.rs

```diff
@@ -323,18 +326,16 @@ impl Order {
         Order {
             id: Uuid::new_v4(),
             maker: MakerId(peer_id),
-            buy: new_order.buy.as_sat(),
-            sell: new_order.sell,
+            pair: TradingPair::BtcDai,
+            direction: new_order.direction,
+            dai_quantity: new_order.dai_quantity,
+            bitcoin_quantity_sat: new_order.bitcoin_quantity.as_sat(),
```

Comment:
Ok, this is starting to look nice. Are we on to 90% review yet? If so, I'd like to see us migrate  to using `btc` instead of `bitcoin`. If we are moving to being asset-centric in cnd this makes more sense and its less characters to type with no loss of meaning.